Makefile

```diff
@@ -36,6 +36,15 @@ docs-lint: ## Check documentation for common syntax errors.
 # The `-n` option enables "nit-picky" mode.
 	make -C docs/ clean && sphinx-build -Wn docs/ docs/_build/html
 
+.PHONY: update-user-guides
+update-user-guides: ## Update screenshots for the user guides.
+	if [ -d "/vagrant" ]; then \
+			bash -c "pushd /vagrant/securedrop; pytest -v tests/pages-layout --page-layout; popd"; \
+			cp /vagrant/securedrop/tests/pages-layout/screenshots/en_US/*.png /vagrant/docs/images/manual/; \
```

Comment:
Nit: the `bash -c` and `cp` lines are indented with three tabs, when they should be indented with two tabs.